src/services/doppler-billing-user-api-client.ts

```diff
@@ -480,7 +480,7 @@ export class HttpDopplerBillingUserApiClient implements DopplerBillingUserApiCli
 
       const response = await this.axios.request({
         method: 'GET',
-        url: `/accounts/${email}/invoices/declined`,
+        url: `/accounts/${email}/invoices?withStatus=2`,
```

Comment:
wath's `2` in `withStatus`?

Reply:
withStatus=2 was the value that represented declinedInvoices, though now it can be changed to withStatus=declined